src/chapter3/shared-fs.md

```diff
@@ -1 +1,54 @@
 # M3's Shared Filesystem
+
+M3 has a unique filesystem consisting of three main important parts (for you).
```

Comment:
Maybe give a general overview about what a shared filesystem is and how it works from a high/abstract level.